jutut/features_and_hints.rst

```diff
@@ -0,0 +1,156 @@
+
+.. hint::
+
+    If you want to send another member of course staff a link directly to a feedback conversation, copy the link from the "Show feedback for this exercise by this student" button.
+
+    .. figure:: /images/jutut/feedback_link.png
+        :width: 100%
+        :alt: The button for getting the link is after the chapter title and the button for filtering feedback for the given exercise.
+
+You as the teacher can respond to student feedback through the Jutut teacher portal.
+The student receives a notification in A+, but also can see the conversation history at the questionnaire embedded in the material. The student can respond by sending another submission.
+
+.. figure:: /images/jutut/conversation_student.png
+    :width: 60%
+
+    *The student perspective of the conversation.*
+
+The responses by the same student to the same questionnaire are grouped together and displayed as a conversation.
+
+.. figure:: /images/jutut/conversation_teacher.png
+    :width: 100%
+
+    *The teacher perspective of the conversation.*
+
+
+Feedback tags
+-------------
+
+Feedback tags allow you to tag feedback conversations with custom tags.
+Feedback responses can be filtered based on these taggings.
+You can create and edit feedback tags for the course instance the their own page (*More* -> *Manage feedback tags*).
+
+.. figure:: /images/jutut/feedbacktags_manage.png
+    :width: 100%
+    :alt: Screenshot of the feedback tags management page titled "List of feedback tags"
+
+
+.. hint:: 
+
+    If you have a lot of feedback tags and use some more often than others and would like them to be "pinned" to the top / displayed before the others, edit the slug so it begins with a small number (e.g. '0' or '1'), as the tags are ordered alphabetically by the slugs.
+
+When filtering using feedback tags, you can select whether the conversations *should have* the specified tag (by clicking once on the tag button) or should *NOT* have the specified tag (by clicking twice on the tag button).
+(The feedback tag buttons in the filter view have three states: (0) inactive ->  (1) include -> (2) exclude -> back to (0) inactive.)
+
+If you select multiple feedback tags to filter using the included criterion, you can specify whether it's enough that the conversations have *any* of the specified tags (use the **OR** operator) or if they should have *all* of the specified tags (use the **AND** operator).
+Despite which operator is being used, if a conversation has any of the tags specified to be excluded, it will not show up in the results.
+
+.. figure:: /images/jutut/feedbacktag_filtering.png
+    :width: 100%
+    :alt: If the user selects that the *DONE* tag is to be excluded and the *Respond* tag to be included in the search, when filtering, conversations that have been tagged with the *Respond* tag and do not have the *DONE* tag will be returned.
+
+    *Filter conversations excluding those with the **DONE** tag and including those with the **Respond** tag.*
```

Comment:
RST does not allow nesting of inline markup so you can't use both bold and italics font at the same time.
https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#inline-markup